### PR TITLE
Feat/backend split issue #50

### LINKED ISSUE #50: Separate Backend
Resolves #40.

### PR DESCRIPTION
**Pull request by basilv**
Splitting API out to its own directory within the repo.

### CODE REVIEW

**Review by DerekRoberts — DISMISSED** [review]

**Review by explorertolu — DISMISSED** [review]

File: api/localdev.ps1 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Environment variable settings needed to run API locally via npm run start:api
```
Comment (DerekRoberts) [inline api/localdev.ps1:1]: Please use env files, not .ps1.  Our only supported platform is Linux.

**Review by DerekRoberts — DISMISSED** [review]

File: api/localdev.ps1 (hunk 1 above)
Comment (DerekRoberts) [inline api/localdev.ps1]: Please use env files, not .ps1.  Our only supported platform is Linux.

Comment (ianliuwk1019) [reply api/localdev.ps1]: Powershell script is removed.

**Review by DerekRoberts — APPROVED** [review]
Hooray!  :)

**Pull request merged**